hdt-lib/include/Iterator.hpp

```diff
@@ -133,6 +133,8 @@ class IteratorTripleID {
 	}
 	virtual void goTo(unsigned int /*pos*/) {
 	}
+	virtual void skip(unsigned int /*pos*/) {
+	}
```

Comment:
I propose to explicitly document the methods above (absolute / relative), so the confusion never happens in the future.